src/build_options.rs

```diff
@@ -68,6 +68,9 @@ pub struct BuildOptions {
     /// Use as `--rustc-extra-args="--my-arg"`
     #[structopt(long = "rustc-extra-args")]
     pub rustc_extra_args: Vec<String>,
+    /// Control whether to build universal2 wheel for macOS or not
```

Comment:
Please add what this does on not-mac platforms

Reply:
Also please add this to the readme (paste the output of `build --help` in the box and maybe a sentence somewhere that there's a universal2 opt-in for macs)